packages/core/src/main/utils/resolve-system-proxy/resolve-system-proxy-window.injectable.ts

```diff
@@ -4,13 +4,21 @@
  */
 import { getInjectable } from "@ogre-tools/injectable";
 import { BrowserWindow } from "electron";
+import electronAppInjectable from "../../electron-app/electron-app.injectable";
 
 const resolveSystemProxyWindowInjectable = getInjectable({
```

Comment:
Name indicates `function`, but it actually returns a `window`. Proposed name: `systemProxyWindowInjectable`.